Daytime-Operations/Auxiliary-Telescope/AT-Calibrations/prepare-atcs-for-flat-fields-procedure.rst

```diff
@@ -8,7 +8,54 @@
 Prepare ATCS For Flat Fields Procedure
 ######################################
 
-Under construction.
+Overview
+========
+
+This procedure describes how to execute flat field calibration with LATISS during day time checkout procedure, using different filters.
+This is the default end-of-checkout script.
+
+It will leave the telescope in position to start taking afternoon calibraitons. This is az = 205.7 elev = 30
+If you are not sure how you want to leave the telescope, consider running this script as it will perform a few more useful checks of the system.
+
+
+Procedure
+==========
+
+Be sure you have run previously the following scripts:
+
+- enable_latiss 
+- latiss_checkout 
+- enable_atcs 
+- atpnuematics_checkout 
+- telescope_and_dome_checkout 
+- slew_and_take_image_checkout 
+
+Setup Telescope and Dome to run the "prepare for flats".
+This is:
+
+- Turn off ATAOS correction
+- Put mirror back on the hardpoints
+- Shut off the valves so we can test the pneumatics
+- Open mirror covers and vents
+- Close mirror covers and vents 
+
+
+Home the dome to make sure it knows where it is.
+Telescope is ready to take calibrations.
```

Comment:
This is the post-condition of your procedure. Now we are ready for calibrations and link to the page to make calibrations https://obs-ops.lsst.io/Daytime-Operations/Auxiliary-Telescope/AT-Calibrations/LATISS-Combined-Calibrations-Procedure/latiss-combined-calibrations-procedure.html